code/controllers/subsystem/assets.dm

```diff
@@ -0,0 +1,23 @@
+var/datum/subsystem/assets/SSasset
+
+/datum/subsystem/assets
+	name = "Assets"
+	priority = 19
```

Comment:
You'll want to set this to a lower priority... other wise assets made by lower priority subsystems won't get sent to world start clients.

in fact, the default would be best.